### PR TITLE
AddBlockRequest message modified

### PR DESCRIPTION
**Pull request by gabriel654165**
added a string id field in addBlockRequest in order to specify a note to add a block into

### CODE REVIEW

File: notes/notes.proto [hunk 1]
```diff
@@ -88,12 +88,15 @@ message ListNotesReply {
 }
 
 message AddBlockRequest {
+    //the note id where we add a block
+    string id = 1;
```
Comment (rojas-diego) [inline notes/notes.proto:92]:
```suggestion
    string note_id = 1;
```

**Review by rojas-diego — CHANGES_REQUESTED** [review]

File: notes/notes.proto (hunk 1 above)
Comment (gabriel654165) [reply notes/notes.proto:92]: Yes right !

File: notes/notes.proto [hunk 2]
```diff
@@ -88,22 +88,31 @@ message ListNotesReply {
...
+    // In order to move the position of an existing block
+    uint32 index = 4;
 
-    google.protobuf.FieldMask update_mask = 3;
+    google.protobuf.FieldMask update_mask = 5;
 }
 
 message DeleteBlockRequest {
-    string id = 1;
+    string note_id = 1;
```
Comment (rojas-diego) [inline notes/notes.proto:115]: Mhm, do we really need this? a `Block.id` is supposed to be unique across all notes right (aka `Block.id` is sufficient to uniquely identify a block)? Only reason why we might need it is to simplify the authorisation process.

File: notes/notes.proto [hunk 3]
```diff
@@ -88,22 +88,31 @@ message ListNotesReply {
...
     // Given that a note is an array of blocks, provide the index at
     // which the block shall be inserted.
-    uint32 index = 2;
+    uint32 index = 3;
 }
 
 message UpdateBlockRequest {
-    string id = 1;
+    // The note id where we add a block
+    string note_id = 1;
```
Comment (rojas-diego) [inline notes/notes.proto:104]: I don't get it, how do you know which block to update if you only pass in a `note_id`. I think we should keep the `id` as it was.

**Comment by rojas-diego** [comment]
Any updates on this?

**Comment by rojas-diego** [comment]
__Please ignore this message__ #17

**Pull request closed without merging**